dcc-import-core/src/main/java/org/icgc/dcc/imports/core/util/Importers.java

```diff
@@ -31,11 +31,18 @@
    * Constants.
    */
   private static final int DEFAULT_MONGO_PORT = 27017;
-  private static final String MONGO_URI_TEMPLATE = "mongodb://localhost:%d/%s";
+  private static final String MONGO_URI_TEMPLATE = makeMongoURI();
+
 
   /**
    * Local MongoDB URIs.
    */
+  private static String makeMongoURI(){
+    if (System.getenv("DOCKER_COMPOSE") !=null )
+        return"mongodb://mongo:%d/%s";
+    else
+        return"mongodb://localhost:%d/%s";
```

Comment:
```suggestion
    return System.getenv("MONGO_DATABASE_URL", "mongodb://localhost:27017/dcc-import");
```

Lets make the mongo db url an environmental variable instead.